docs/debug-kernel.md

```diff
@@ -0,0 +1,45 @@
+# Get Clear Containers Kernel logs
+
+Clear Containers runtime launches a special virtual machine when a new pod is
```

Comment:
- I think this should be "Intel® Clear Containers", but maybe you could avoid that by just writing, "`cc-runtime` launches...".
- I'd drop "special".

Reply:
Done.